src/fuel-vm/instruction-set.md

```diff
@@ -2473,6 +2473,8 @@ Get [fields from the transaction](../tx-format/transaction.md).
 | `GTF_SCRIPT_INPUT_AT_INDEX`               | `0x00B` | Memory address of `tx.inputs[$rB]`                                |
 | `GTF_SCRIPT_OUTPUT_AT_INDEX`              | `0x00C` | Memory address of `t.outputs[$rB]`                                |
 | `GTF_SCRIPT_WITNESS_AT_INDEX`             | `0x00D` | Memory address of `tx.witnesses[$rB]`                             |
+| `GTF_TX_START_ADDRESS`                    | `0x0A0` | Memory address of the raw transaction bytes                         |
+| `GTF_TX_LENGTH`                           | `0x0A1` | Length of raw transaction types in memory                        |
```

Comment:
Why do we want to put size and offset under `0x0A1` and `0x0A0`? Maybe it is better to put them at the end

Reply:
No real reason. The location just seemed appropriate. I'll move them. deb35bca